setup.cfg

```diff
@@ -100,6 +100,58 @@ remote_data_strict = True
+    # add a comment or ideally a link to an issue that explains why the warning
+    # is being ignored
+    #
+    #
+    # See https://github.com/pandas-dev/pandas/issues/26367; this can be
+    # removed when our minimum pandas version is >=0.25
+    ignore:can't resolve package from __spec__ or __package__:ImportWarning
+    # See https://github.com/spacetelescope/asdf/issues/789
+    ignore:direct construction of AsdfSchemaFile has been deprecated
+    ignore:direct construction of AsdfSchemaItem has been deprecated
+    ignore:direct construction of AsdfSchemaExampleItem has been deprecated
+    # See https://github.com/Cadair/parfive/issues/41
+    ignore:The loop argument is deprecated
+    # This is due to dependencies building with a numpy version different from
+    # the local installed numpy version, but should be fine
+    # See https://github.com/numpy/numpy/issues/15748#issuecomment-598584838
+    ignore:numpy.ufunc size changed:RuntimeWarning
+    ignore:numpy.ndarray size changed:RuntimeWarning
+    # See https://github.com/mvantellingen/python-zeep/issues/956
+    ignore:defusedxml.lxml is no longer supported:DeprecationWarning
+    # See https://github.com/aio-libs/aiohttp/issues/4842
+    ignore:"@coroutine" decorator is deprecated
+    # See https://github.com/spacetelescope/asdf/issues/789
+    ignore:direct construction of AsdfSchemaFile has been deprecated
+    ignore:direct construction of AsdfSchemaItem has been deprecated
+    ignore:direct construction of AsdfSchemaExampleItem has been deprecated
+    # See https://github.com/scikit-image/scikit-image/issues/4848
+    ignore:Converting `np.inexact` or `np.floating` to a dtype is deprecated
+    # See https://github.com/astropy/astropy/pull/10570
+    ignore:leap-second auto-update failed
+    # See https://github.com/astropy/extension-helpers/issues/23
+    ignore:Distutils was imported before Setuptools
+    # Figure tests use agg backend so show doesn't work but we use it in peek
+    ignore:Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.
+    # Ignore doctest issues: docs/guide/tour.rst
+    ignore:File may have been truncated*
+    # Some tests use data that have dates in the future and ERFA does not like.
+    ignore:ERFA function "d2dtf"*
+    ignore:ERFA function "dtf2d"*
+    # Currently helio makes unverified requests - this filter should be removed when
+    # https://github.com/sunpy/sunpy/issues/4401 is fixed
+    ignore:Unverified HTTPS request
+    # https://github.com/astropy/astropy/issues/11272
+    ignore:elementwise == comparison failed and returning scalar instead:FutureWarning:astropy.table
+    # This isn't really an issue it's an annoyance, it's triggered by printing
+    # an astropy table inside a test, which is generally only done when
+    # debugging.
+    ignore:the imp module is deprecated in favour of importlib:DeprecationWarning:ipykernel.iostream
```

Comment:
Why add these lines? We're still ignoring warnings on the 2.0.x branch.

Reply:
It came with a backport and I cba to remove it.

Reply:
Removed.